docs/ibexa_products/headless.md

```diff
@@ -0,0 +1,180 @@
+  But, as a framework, more complex workflows can be imagined, with several steps and paths, even some automated tasks.
+
+#### [[= product_name_headless =]] accelerates content editing
+
+- [[= product_name_headless =]]'s content tree has several actions available directly on its items. For example, no need to open a content to hide it, you can do it directly from the content tree.
+- An Image Editor offers to crop and flip images. When serving the image in various context, you can even set a focal point to indicate to automated cropping which part of the image should be kept.
+- A Digital Asset Management (DAM) help to crawl in your image ressources to use and reuse them in your content. And a DAM connector allows to also search in your images hosted on third party DAM servers.
+
+#### [[= product_name_headless =]] integrates into the networks
+
+##### Intranets and extranets
+
+- [[= product_name_connect =]] is to create application interconnections with low code, drag-and-drop and a very visual interface.
+  Complex data flows can be easily implemented with a huge library of connectors and actions for famous to specific applications.
+  See [[[= product_name_connect =]] product guide]([[= connect_doc =]]/general/ibexa_connect/) for more.
+- An OAuth 2 server offers the possibility to use the DXP as the authentification service for other applications.
+- An OAuth 2 client supports authentication with a third-party OAuth 2 server.
+- A DAM Connector, previously mentioned, helps to access any image repository when needing to illustrate a content.
+- [[= product_name_headless =]] supports Elasticsearch and Solr.
+  It gives the choice between using Solr or Elasticsearch as a search engine, whether hosted on [[= product_name_cloud =]] or on-premises.
+  This choice might be influenced by technology you already use, or you want to invest in for other internal projects.
+- [[= product_name_headless =]] offers to export and import from command line part of the content model or content items.
+  For example, it can be used to move new content types and items from a staging instance to the production one.
+
+##### Internet, delivery, web search engines, and social networks
+
+- [[= product_name_headless =]] comes with the support of Fastly content delivery network (CDN).
+  The HTTP cache varies on current user's role and is purged when content change.
+  With is huge network of points of presence (POP) around the world, Fastly is quickly delivering cached content from nearest server for a better user experience.
+- A Search Engine Optimization (SEO) field implements best practices about web search engine indexing and social network sharing.
+  It covers canonical URLs which are mandatory if multiple locations are used for a same Content item to avoid duplicate content,
+  Open Graph protocol to better describe a Content item to social networks and search engine, and Twitter Cards.
+
+## Use cases
+
```

Comment:
I'd turn this list onto an intro paragraph: choose two, three most compelling use cases from the list, create a sentence or two with them, followed by the two paragraphs that you already have.

Reply:
I moved few things to the two example and just turn this into a section intro.